oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -851,6 +895,10 @@ public String getQuotaProjectId() {
       return quotaProjectId;
     }
 
+    int getLifetime() {
```

Comment:
The other Builder getters are public, this should be ok to make public too as it is a public user option.

Reply:
added `public` back